lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -46,9 +65,19 @@ all_afs:
   config_get_token_append: '/^address-family (\S+) (\S+)$/'
 
 client_to_client:
-  config_get_token_append: '/^client-to-client reflection$/'
-  config_set_append: '<state> client-to-client reflection'
-  default_value: true
+  cli_nexus:
+    config_get_token_append: '/^client-to-client reflection$/'
+    config_set_append: '<state> client-to-client reflection'
+    default_value: true
+  cli_ios_xr:
+    config_set: 'router bgp <asnum>'
+    config_set_append:
+      - ' address-family <afi> <safi>'
+      - ' <state> bgp client-to-client reflection disable'
```

Comment:
The commands for nexus and iOS-xr are different and appear at different levels.
Even the defaults are different for client-to-client reflection.


> On 17 Nov 2015, at 16:58, Glenn Matthews <notifications@github.com> wrote:
> 
> In lib/cisco_node_utils/cmd_ref/bgp_af.yaml <https://github.com/cisco/cisco-network-node-utils/pull/87#discussion_r45077538>:
> 
> > @@ -46,9 +65,19 @@ all_afs:
> >    config_get_token_append: '/^address-family (\S+) (\S+)$/'
> >  
> >  client_to_client:
> > -  config_get_token_append: '/^client-to-client reflection$/'
> > -  config_set_append: '<state> client-to-client reflection'
> > -  default_value: true
> > +  cli_nexus:
> > +    config_get_token_append: '/^client-to-client reflection$/'
> > +    config_set_append: '<state> client-to-client reflection'
> > +    default_value: true
> > +  cli_ios_xr:
> > +    config_set: 'router bgp <asnum>'
> > +    config_set_append:
> > +      - ' address-family <afi> <safi>'
> > +      - ' <state> bgp client-to-client reflection disable'
> You're doing this to exclude VRF support for this property? Not sure this is the right way to address that - perhaps @RichWellum <https://github.com/RichWellum> can comment?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cisco/cisco-network-node-utils/pull/87/files#r45077538>.
>